### PR TITLE
Added questions

### PR DESCRIPTION
**Pull request by mihai-db**
Images

### CODE REVIEW

File: aws/aws-quiz.md [hunk 1]
```diff
@@ -261,3 +261,313 @@ FlowLog:
...
+- [x] Lambda is stateless, so it won't remember who a user is in between requests
+
+#### Q20. A principle of DevOps is to view infrastructure as code. Which AWS service allows you to script your AWS infrastructure?
+- [ ] CloudTrail
+- [x] CloudFormation
+- [ ] AWS Config
+- [ ] AWS Service Catalog
+
+#### Q21. You created a Windows EC2 instance with a public IP address and installed SQL Server. When attempting to connect to SQL Server from SQL Server Enterprise Manager on your local computer, the Windows EC2 instance is unable to establish a connection to the server. What is the first thing you should check?
+- [ ] Check the routing tables for the VPC.
```
Comment (Ebazhanov) [inline aws/aws-quiz.md:420]: @mihai-db FYI I think it is all duplication, please double-check.

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Hello, @mihai-db! Thanks for another PR.
Left a comment about duplication also I saw you added a lot of images, but there is no reference for them inside the `aws-quiz.md` file? If you want just keep them also good, but would be great if you could link them to the proper question. If you need help pls let me know.

**Comment by rradfar** [comment]
I went through the 10 images that were included in this PR. They are screenshots of actual assessment questions so we probably do not want them like that in our codebase. That being said these are new questions which we do not have so it would really cool if @mihai-db could add them or create an issue and put them there. 

None of the images are related to AWS (which is what this PR is for). So I suggest opening separate PRs for them. Here is a list of the screenshots and where they belong:
- JSON: 2 questions
- MongoDB: 2 questions
- MySQL: 1 question
- Swift: 5 questions

**Comment by mihai-db** [comment]
> I went through the 10 images that were included in this PR. They are screenshots of actual assessment questions so we probably do not want them like that in our codebase. That being said these are new questions which we do not have so it would really cool if @mihai-db could add them or create an issue and put them there.
> 
> None of the images are related to AWS (which is what this PR is for). So I suggest opening separate PRs for them. Here is a list of the screenshots and where they belong:
> 
> * JSON: 2 questions
> * MongoDB: 2 questions
> * MySQL: 1 question
> * Swift: 5 questions

Could you please do it? My time is lacking here.

**Comment by rradfar** [comment]
> Could you please do it? My time is lacking here.

Sure, no worries buddy.

**Comment by Ebazhanov** [comment]
> > Could you please do it? My time is lacking here.
> 
> Sure, no worries buddy.

What is the plan @rradfar?

**Comment by rradfar** [comment]
> What is the plan @rradfar?

@Ebazhanov I created 5 new issues and captured all the screenshots that @mihai-db has posted here in those tickets. Either myself or other contributors will go through them and add them to our code base. 

I am currently working on updating and cleaning up the `aws-quiz.md` file. I will create a new PR once it is ready and then we will be able to close this PR.

**Comment by PAODLT** [comment]
Hi I help to answer outlook questions, but you didn't accept my request? You think my answer are wrong?

**Comment by rradfar** [comment]
@PAODLT Hi there! You are commenting on a pull request that is not related to your question. May I suggest [creating a discussion topic](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/discussions) or contacting the [project maintainer](https://github.com/Ebazhanov) directly? 

Here is a list of [all open, merged, and closed pull requests](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pulls?q=) and I don't see any pull requests from you. I can see [your code in your repo](https://github.com/PAODLT/linkedin-skill-assessments-quizzes/commit/898907d356e5d12d86e3130de8fd8c5f0ce41e62#diff-353d29da0e2beee411bc639fa1436a3f6892be555422b95db5d7fc6cca8a7851) and it looks great. Could you please [create a pull request](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/compare) so that we can verify your contribution and merge it with our codebase? Let us know if there are any issues.

**Comment by mihai-db** [comment]
> > What is the plan @rradfar?
> 
> @Ebazhanov I created 5 new issues and captured all the screenshots that @mihai-db has posted here in those tickets. Either myself or other contributors will go through them and add them to our code base.
> 
> I am currently working on updating and cleaning up the `aws-quiz.md` file. I will create a new PR once it is ready and then we will be able to close this PR.

Thanks!

**Comment by rradfar** [comment]
@mihai-db Thanks for the new GCP slides. We can't merge them into the codebase like that so I captured your screenshots in issue #1218 to be added by one of the contributors to the [gcp-quiz.md](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/master/google-cloud-platform/gcp-quiz.md) file. 

If you find any new screenshots or problems, feel free to add them directly to the [Issues section](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/issues) of this project by creating a new issue. You don't need to commit them to the codebase, if they are images you can just drag and drop them into a new issue. Thanks.

**Pull request closed without merging**